src/SonarScanner.MSBuild.TFS/CoverageReportProcessorBase.cs

```diff
@@ -36,7 +36,7 @@ public abstract class CoverageReportProcessorBase : ICoverageReportProcessor
         private IBuildSettings settings;
         private string propertiesFilePath;
 
-        private bool succesfullyInitialised = false;
+        private bool successfullyInitialised;
```

Comment:
Optional grammar:
```suggestion
        private bool successfullyInitialized;
```
eventually, it should be fixed in the whole file

Reply:
British English vs US one :)

Reply:
Handled by https://github.com/SonarSource/sonar-scanner-msbuild/pull/1784